ARC specification.md

```diff
@@ -232,6 +233,37 @@ The file `arc.cwl` MUST exist at the root directory of each ARC. It describes wh
 
 `arc.cwl` MUST be a CWL v1.2 workflow description and adhere to the same requirements as [run descriptions](#run-description). In particular, references to study or assay data files, nested workflows MUST use relative paths. An optional file `arc.yml` MAY be provided to specify input parameters.
 
+### Data Path Annotation
+
+All metadata references to files or directories located inside the ARC MUST follow the following patterns:
+
+- The `general pattern`, which is universally applicable and SHOULD be used is to specify the path relative to the ARC root
+
+- The `folder specific pattern`, which MAY be used. This pattern dependes on the metadata context:
```

Comment:
which MAY be used <in the following circumstances> ?

Reply:
notsure about the wording but i think the sentence currently could imply that it MAY be used all the time, even if you specify cases afterwards

Reply:
I collapsed the two sentences into one, hopefully explicating the constraint